apple-pie/index.html

```diff
@@ -1,21 +1,57 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Apple Pie Recipe</title>
-    <!-- don't forget to link your styles -->
-  </head>
-  <body>
-    Apple Pies This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
-    be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
-    favorites as well! Ingredients 1 recipe pastry for a 9 inch double crust pie 1/2 cup unsalted butter 3 tablespoons
-    all-purpose flour 1/4 cup water 3 tablespoons all-purpose flour 1/4 cup water 1/2 cup white sugar 1/2 cup packed
-    brown sugar 8 Granny Smith apples - peeled, cored and sliced Directions Preheat oven to 425 degrees F (220 degrees
-    C). Melt the butter in a saucepan. Stir in flour to form a paste. Add water, white sugar and brown sugar, and bring
-    to a boil. Reduce temperature and let simmer. Place the bottom crust in your pan. Fill with apples, mounded
-    slightly. Cover with a lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-    that it does not run off. Bake 15 minutes in the preheated oven. Reduce the temperature to 350 degrees F (175
-    degrees C). Continue baking for 35 to 45 minutes, until apples are soft.
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="./styles/style.css">
+  <!-- don't forget to link your styles -->
+</head>
+
+<body>
+  <header>
+    <div class="bkgimg">
+      <h1>Apple Pies</h1>
+    </div>
+  </header>
+
+  <section id="presentation">
+    <p>This was my grandmother's apple pie recipe. I have never seen another one quite like it. It will always
+      be my favorite and has won me several first place prizes in local competitions. I hope it becomes one of your
+      favorites as well!</p>
+    <img src="./images/recipe-info.png" alt="recipe-info" id="recipe">
+    <hr>
+  </section>
+
+
+  <section id="ingredients">
+    <h2 class="titulo">Ingredients</h2>
+    <br>
+    <p>
+      - 1 recipe pastry for a 9 inch double crust pie<br>
+      - 1/2 cup unsalted butter<br>
+      - 3 tablespoons all-purpose flour<br> 
+      - 1/4 cup water<br>
+      - 1/2 cup white sugar<br>
+      - 1/2 cup packed brown sugar<br>
+      - 8 Granny Smith apples - peeled, cored and sliced<br>
+    </p>
```

Comment:
I know you know, but here a < ul > would be more adapted 😉 
To have the "-" bullet style, you can do 
`ul li:before {
  content: "- ";
}
`